docs/DeveloperGuide.md

```diff
@@ -323,14 +323,60 @@ and the **Actor** is the `user`, unless specified otherwise)
 
       Use case resumes at step 2.
 
+**Use case: Delete job**
+
+**MSS**
+
+1. User requests to list jobs
+2. TalentConnect shows a list of jobs
+3. User requests to delete a specific job in the list
+4. TalentConnect deletes the job and unmatches all contacts matched to the job
+5. TalentConnect returns an updated list of jobs and contacts along with a success message
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. The job list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+     * 3a1. TalentConnect shows an error message.
+
+       Use case resumes at step 2
+
+**Use case: Delete company**
+
+**MSS**
+
+1. User requests to list companies
+2. TalentConnect shows a list of companies
+3. User requests to delete a specific company in the list
+4. TalentConnect deletes the company and deletes all jobs associated with the company
```

Comment:
It is probably good to explicitly mention that contacts will be "unmatched".

Reply:
ok